src/main/resources/templates/fragments/reusable-components.html

```diff
@@ -1,43 +1,59 @@
-<div th:fragment="navbar">
-    <nav class="navbar navbar-dark bg-primary">
-        <div class="container-fluid">
-            <a class="navbar-brand">Hobby Collection</a>
-            <form class="d-flex">
-                <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-                <button class="btn btn-outline-success" type="submit">Search</button>
-            </form>
-        </div>
-    </nav>
-</div>
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
```

Comment:
I fixed the error "Namespace 'th' is not bound".
Sources:
https://stackoverflow.com/questions/32384158/intellij-thymeleaf-namespace-th-is-not-bound

Reply:
This is a fragment, though... the th may be in the enclosing page.